Functions/GeoTrack.ps1

```diff
@@ -101,6 +101,25 @@ Function Convert-GeoTrack {
     }
 }
 
+Function Get-GoogleSessionVariable {
+    [CmdletBinding()] param(
+        [string] $email,
+        [string] $password

```

Comment:
CredentialManagerCredential.ps1 (horrible name) provides a means for saving credentials into credential manager.  I suggest we take a dependency on this so that credentials are not always required.